### ISSUE TITLE
Groovy is broken?

### ISSUE THREAD

**Issue by zenfyrdev** [elect86/magik#7]
Hi, the plugin looks interesting, but it seems broken when using groovy.

```groovy
plugins {
    id 'maven-publish'
    id "elect86.magik" version "0.3.3"
}

repositories {
    mavenCentral()
    githubPackages("kotlin-graphics/mary")
    maven { url 'https://jitpack.io' }
}
```

Log: https://gist.github.com/melontini/3ec6f1f7271f33298395e4fd7509091a

**Comment by zenfyrdev** [comment]
Oh, alright, there's a workaround using auth url

```groovy
maven {
        url = "https://elect86:ghp_${new String(Base64.decoder.decode("N1YxWWxqRWNLU2h6d1B6UEp1QVBQMFg1NXVyRWhGMFJCV0cy"))}@maven.pkg.github.com/kotlin-graphics/mary"
    }
 ```

**Comment by elect86** [comment]
I'm not a Groovy expert, but I'm totally open to whatever would make Groovy life users easier

If you happen to know, don't hesitate, I can also send you an invitation

**Comment by zenfyrdev** [comment]
I honestly have no idea what I'm doing when it comes to groovy/gradle, so I'm not much of help here.

**Comment by elect86** [comment]
Which project is? Would you consider switching it to Kotlin script?

**Comment by KorewaLidesu** [comment]
I believe this is what you want to work with if you want to use Github Packages in Gradle
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry

**Comment by zenfyrdev** [comment]
Yeah, I just opted for a remote apply script which gets the credentials from the `credentials` file.

**Comment by KorewaLidesu** [comment]
Oh....
I see what you are try to do here.
The issue you are getting is try to make the Github repo work huh
Try this instead?
```
dependencies {
    implementation 'com.example:package'
}
```
